### ISSUE TITLE
Network fails attempting to visualize LSTM

### ISSUE THREAD

**Issue by dsblank** [Calysto/conx#17]
From notebook/LSTM.ipynb

```python
print('Creating Model...')
net = Network("LSTM")
net.add(Layer("input", (1, 1), batch_shape=(25, 1, 1))) 
net.add(LSTMLayer("lstm1",
                  50,
                  batch_size=25,
                  return_sequences=True,
                  stateful=True))
net.add(LSTMLayer("lstm2",
                  50,
                  return_sequences=False,
                  stateful=True))
net.add(Layer("output", 1))
net.connect()
net.compile(loss='mse', optimizer='rmsprop')
net
```

**Comment by dsblank** [comment]
Need to take account of batch_shape, or get it from a Sequence() object when loading the dataset. This will propagate in the above network:

```python
x = np.array([[[0.5]]] * 25)
print(x.shape)
net.model.predict(x)
```

**Comment by dsblank** [comment]
Fixed in 3.3.4. See example notebooks.
